xgboost_bench/README.md

```diff
@@ -1,56 +0,0 @@
-# How to create conda environment for benchmarking
```

Comment:
Can't find where XGBoost/MB benchmarking beeping moved?  where they are in new setup?